android/src/main/java/com/sketchView/SketchView.java

```diff
@@ -47,6 +47,10 @@ public void setToolType(int toolType) {
         }
     }
 
+    public void setPenColor(int toolColor) {
+        currentTool = new PenSketchTool(this, toolColor);
```

Comment:
should check if current tool implements `ToolColor` interface, and calls the `setToolColor` on the current tool if it does.